platform/core/src/services/ColorbarService/ColorbarService.ts

```diff
@@ -0,0 +1,146 @@
+import { PubSubService } from '../_shared/pubSubServiceInterface';
+
+const EVENTS = {
+  COLORBAR_STATE_CHANGED: 'event::ColorbarService:stateChanged',
```

Comment:
I mean this is getting called like `ColobarService.Events` so I guess it is redundant to say ColorbarService.Events.COLORBAR_STATE_CHANGED ? Maybe just STATE_CHANGED?
Make the Event an static proerties of the class see others

Reply:
I agree, done.